<!-- %Tag(FULL)%-->
<launch>

 <!-- Turtlesim Node-->
  <node pkg="turtlesim" type="turtlesim_node" name="sim"/>


 <!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="turtle_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
    <param name="autorepeat_rate" value="5"/>
  </node>

 <!-- Axes -->
  <param name="axis_force" value="1" type="int"/>
  <param name="axis_torque" value="2" type="int"/>
  <param name="force_scale" value="2" type="double"/>
  <param name="torque_scale" value="2" type="double"/>

  <node pkg="learning_joy" type="joystick_wrench" name="joy_wrench"/>

</launch>
<!-- %EndTag(FULL)%-->
